Bio/PDB/parse_pdb_header.py

```diff
@@ -103,8 +103,7 @@ def _nice_case(line):
         if c >= 'a' and c <= 'z' and nextCap:
             c = c.upper()
             nextCap = 0
-        elif c == ' ' or c == '.' or c == ',' or c == ';' or c == ':' or c == '\t' or\
-             c == '-' or c == '_':
+        elif c in ' .,;:\t-_':
```

Comment:
@etal @JoaoRodrigues @lennax could one of you OK this refactor in the PDB header parsing code?

(I wanted to fix the indentation warning by avoiding the slash-based line continuation, but going further this version seems more elegant)

Reply:
This looks good to me. Much cleaner.

Reply:
Thanks - I was pretty sure I had the same list of symbols, but doubt we had 100% coverage here, so a second set of eyes on this is appreciated.